### PR TITLE
[Samples][13.core-bot] Add deployment scripts

### PR DESCRIPTION
**Pull request by FedericoBernal**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

**Review by Batta32 — APPROVED** [review]
LGTM, but I'll wait to merge this as I want to validate with Tracy if we really need to migrate this

File: samples/13.core-bot/deploymentScripts/linux/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+if [ "x$CLEAN_LOCAL_DEPLOYMENT_TEMP" = xtrue ]; then
+    rm -rf "$DEPLOYMENT_TEMP"
+    mkdir "$DEPLOYMENT_TEMP"
+fi
+
+##################################################################################################################################
+# Deployment
+# ----------
+
+echo Handling ASP.NET Core Web Application deployment.
```
Comment (matiasroldan6) [inline samples/13.core-bot/deploymentScripts/linux/deploy.sh:81]: this mention to ASP.NET is ok?

Comment (FedericoBernal) [reply samples/13.core-bot/deploymentScripts/linux/deploy.sh:81]: nope, you're right. I'll change that

File: samples/13.core-bot/deploymentScripts/linux/deploy.sh [hunk 2]
```diff
@@ -78,7 +78,7 @@ fi
 # Deployment
 # ----------
 
-echo Handling ASP.NET Core Web Application deployment.
+echo Handling Java Core Web Application deployment.
```
Comment (matiasroldan6) [inline samples/13.core-bot/deploymentScripts/linux/deploy.sh:81]: I think the part to go was the entire "ASP.NET Core"

**Review by matiasroldan6 — APPROVED** [review]

**Pull request closed without merging**